proto/regen/ecocredit/v1beta1/state.proto

```diff
@@ -0,0 +1,213 @@
+syntax = "proto3";
+
+package regen.ecocredit.v1beta1;
+
+import "cosmos/orm/v1alpha1/orm.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+// ClassInfo represents the high-level on-chain information for a credit class.
+message ClassInfo {
+
+    option (cosmos.orm.v1alpha1.table) = {
+        id: 1,
+        primary_key: {fields: "class_id"}
```

Comment:
Maybe let's add admin and credit type indexes?

Reply:
https://github.com/regen-network/regen-ledger/pull/700/files#diff-302ae7137e2a8ea4a04301cf72f3df8311c99ea4716efaf59ae04291e030fb63R16-R17